csharp/ql/src/semmle/code/csharp/dataflow/CallContext.qll

```diff
@@ -46,25 +45,22 @@ abstract class ArgumentCallContext extends CallContext {
    * Holds if this call context represents the argument at position `i` of the
    * call expression `call`.
    */
-  abstract predicate isArgument(DotNet::Expr call, int i);
+  abstract predicate isArgument(Expr call, int i);
```

Comment:
I guess we don't bother with `CIL::Expr` because we don't analyse virtual dispatch in CIL anyway. So it makes sense to remove these from the call context for now.

Reply:
Actually, this is because these call contexts are now only used when resolving delegate targets in `DelegateDataFlow.qll`, and there flow is restricted to C# code. The shared `DataFlowImplCommon.qll` has its own notion of a call context.